src/java_bytecode/java_class_loader.cpp

```diff
@@ -31,31 +31,35 @@ unsigned char java_core_models[] = { JAVA_CORE_MODELS_DATA };
 java_bytecode_parse_treet &java_class_loadert::operator()(
   const irep_idt &class_name)
 {
-  std::stack<irep_idt> queue;
+  std::stack<irep_idt> stack;
```

Comment:
I realise it was always a stack and not as the name suggested a queue, but would a std::queue actually make a lot more sense?